arr-lib.js

```diff
@@ -0,0 +1,110 @@
+var arrLib = {};
+
+arrLib = (function (context) {
+
+    context.max = function (arr) {
+        return min_max(arr, "max");
+    };
+
+    context.min = function (arr) {
+        return min_max(arr, "min");
+    };
+
+    let min_max = function (arr, type) {
```

Comment:
1. It's not great idea to write common method to min and max utils: you increase amount of code lines and you write `switch` statement (using of `switch` usually is bad practice)
2. You may just use methods `Math.max` and `Math.min`